de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/ToolAdminDetailsDialog.java

```diff
@@ -94,24 +86,9 @@ public void show(final Tool tool, Mode mode) {
 
     @Override
     public void show() {
-        final ToolContainer toolContainer = factory.getContainer().as();
-        toolContainer.setDeviceList(Lists.<ToolDevice>newArrayList());
-        toolContainer.setContainerVolumes(Lists.<ToolVolume>newArrayList());
-        toolContainer.setContainerVolumesFrom(Lists.<ToolVolumesFrom>newArrayList());
-        ToolImage image = factory.getImage().as();
-        image.setName(appearance.defaultImgPrefix());
-        toolContainer.setImage(image);
-
-        final ToolImplementation toolImplementation = factory.getImplementation().as();
-        final ToolTestData toolTestData = factory.getTest().as();
-        toolTestData.setInputFiles(Lists.<String>newArrayList());
-        toolTestData.setOutputFiles(Lists.<String>newArrayList());
-        toolImplementation.setTest(toolTestData);
```

Comment:
Yeah, sounds ok to me.  I think this is the only spot where we set up the whole JSON structure so... makes sense to leave it there.